shell/browser/browser.h

```diff
@@ -137,7 +137,7 @@ class Browser : public WindowListObserver {
 #if defined(OS_WIN)
     // used in browser::setLoginItemSettings
     bool enabled = true;
-    std::u16string name = std::u16string();
+    std::wstring name = std::wstring();
```

Comment:
```suggestion
    std::wstring name;
```

no need to explicitly call the constructor here.

Reply:
Removed 0f3620099a3de57956370e3c463d02bd0b2dc824